### ISSUE TITLE
Set of documentation Updates 

### ISSUE THREAD

**Issue by jperk224** [AndBible/docs#3]
New to the app and walking through documentation.  Making note of some updates I'd like to make as I go.

- [ ] Getting Started > Changing the Look and Feel - fix verbiage to match updated app UI
- [ ] Backup and Restore > How to restore your data - remove 2. 'Tap on Restore Documents From' under restoring modules, dictionaries, commentaries; menu now goes directly to step 3
- [ ] Bookmarks > Creating a Bookmark > adding to part of a verse - Change 3 to Choose _Bookmark_ > _Selection_; with note about option to bookmark the entire verse; Change 4 to _Select or edit labels..._

**Comment by jperk224** [comment]
I'd like to add to this list/work on these as I traverse the docs...
I did not see docs on the workflow for this repo concerning guidelines about opening/working issues.  Let me know if there are specifics to follow.  Thanks!

**Comment by timbze** [comment]
Hi @jperk224, so far there are no specifics really. Feel free to make PRs for documentation improvements or ask questions you may have. We greatly appreciate the help.

**Comment by agrogers** [comment]
Yes, the docs are very rough at the moment. Maybe we can include them in the app itself one day. But providing something for every language is a big task which is unlikely to ever get done.

**Comment by jperk224** [comment]
Is there a trick to getting the build in readthedocs to reflect your changes? I got through the tutorial without issue.  I forked this repo and tried a test PR and none of my changes were reflected in the docs...

1. I do not see a `.yml` configuration file, so I _assume_ the admin settings take precedence?  When I created a test PR I expected the build to reflect its changes (I made a test change to `index.rst`) like what happened in the tutorial because I had enabled `Build Pull Requests for the project`; But my PR changes were not reflected... Is there overriding configuration I am missing?
2. So I created a sandbox mirror of this repo set it up fully in readthedocs and made some changes to `index.rst`, merged the PR into main and performed a build.  But my changes still aren't reflected when I `View Docs`.  My changes are in `index.rst` in my sandbox repo, and the doc rebuilt without errors, but my changes aren't reflected in the docs?

I feel like I'm missing a step?  Any insight is appreciated.  I re-synced my github webhook and it appears to be connected.  When I create a PR I get the notification on the PR that readthedocs build is in progress, but then when I view the completed build, my changes are not there.

Thanks!

**Comment by jperk224** [comment]
Oh I am so sorry. Please disregard all of that above... it was a browser cache issue. My apologies...

**Comment by agrogers** [comment]
Glad to hear you got it working
